app/styles/page-not-found.css

```diff
@@ -0,0 +1,13 @@
+.page-not-found{
+    margin: auto;
+    text-align: center;
+  }
+  
+  .page-not-found__img {
+    width: 20%;
+  }
+  
+  .page-not-found__text{
+    font-size: 15px;
+    font-weight: bold;
+  }
```

Comment:
Can we please fix the formatting here?
Apologies for not configuring the pre-commit hook. I will do it, then linting issues can be prevented